src/main/java/com/epam/ta/reportportal/core/launch/impl/GetLaunchHandlerImpl.java

```diff
@@ -226,7 +226,7 @@ private Iterable<LaunchResource> getLaunchResources(Page<Launch> launches) {
 
 	@Override
 	public List<String> getLaunchNames(ReportPortalUser.ProjectDetails projectDetails, String value) {
-		expect(value.length() >= MIN_LAUNCH_NAME_LENGTH && value.length() <= MAX_LAUNCH_NAME_LENGTH, equalTo(true)).verify(
```

Comment:
Why we removing MIN_LAUNCH_NAME_LENGTH here? And if we need to remove it, then you also should remove this from log message

Reply:
Will be fixed, thx